tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -1,7 +1,12 @@
 import os
 import subprocess
 
-from conftest import SYSTEM_RELEASE_ENV
+from os import getenv
```

Comment:
We can take care of the codeql notice in https://github.com/oamg/convert2rhel/pull/740